package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "wookone",
   "version": "0.1.0",
-  "private": true,
+  "private": false,
```

Comment:
FYI, this only needs to be `false` if you want to publish the project to the NPM registry: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private

Since this is just an app you'll publish to Netlify or wherever, it might as well stay private.

Reply:
ok that's interesting to know. thanks. @dahlbyk